website/templates/base.mako

```diff
@@ -189,7 +185,11 @@
                     timezone: ${ user_timezone | sjson, n },
                     entryPoint: ${ user_entry_point | sjson, n },
                     institutions: ${ user_institutions | sjson, n},
-                    emailsToAdd: ${ user_email_verifications | sjson, n }
+                    emailsToAdd: ${ user_email_verifications | sjson, n },
+                    anon: {
```

Comment:
See my previous comment re: nested, namespaced representations. If `get_globals` is changed, this can be simplified to

```js
anon: ${ anon | sjson, n}
```